deployments/local-scripts/relay-messages-rialto-to-millau.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# A script for relaying Rialto messages to the Millau chain.
+#
+# Will not work unless both the Rialto and Millau are running (see `run-rialto-node.sh`
+# and `run-millau-node.sh).
+set -xeu
+
+MILLAU_PORT="${MILLAU_PORT:-9945}"
+RIALTO_PORT="${RIALTO_PORT:-9944}"
```

Comment:
I think it would be more clear from a `.env` file point of view to call these `NETWORK_WS_PORT`